tasks/main.yml

```diff
@@ -22,11 +22,31 @@
   - include_tasks: release.yml
     when: rhsm_release or rhsm_release_unset
 
+  # decide whether or not to use the vendored rhsm_repository module,
+  # which first appeared in ansible 2.5
+  - name: Determine if vendored rhsm_repository module should be used
+    set_fact:
+      # If you use the | or > line-folding syntax, the result is forced to be a
+      # string. Since this fact really needs to be a bool, this task is forced
+      # to have a long line.
+      # yamllint disable-line rule:line-length
+      _rhsm_use_vendored_repository: "{{ ansible_version.full is version_compare('2.5', '<') | bool }}"
```

Comment:
I'm fairly sure we could simplify this and the other two related set_facts with a ternary, but I'm not sure I want to.
Talk me into it. Or out of it. :)

Reply:
Something like this might work
```yaml
_rhsm_repository_task: "{{ ansible_version.full is version_compare('2.5', '<') | ternary('rhsm_repository', '_rhsm_repository') }}"
```
Can probably deal with the linting using a line fold, too:
```yaml
_rhsm_repository_task: >
  {{ ansible_version.full is version_compare('2.5', '<') |
  ternary('rhsm_repository', '_rhsm_repository') }}"

Reply:
I may have just talked myself into it.

Reply:
Do that thing. It's a better thing.